### PR TITLE
fixes issue #86

### PR DESCRIPTION
**Pull request by richpsharp**
This specifically tests the log level against the message level, updates the test that should have covered this bug.

### CODE REVIEW

**Comment by richpsharp** [comment]
hi @phargogh, i tried to do this right, but it looks like all the tests failed, but not because of this change... like flake8 failed on `AttributeError: 'EntryPoints' object has no attribute 'get'` and python 3.6 failed on `Version 3.[6](https://github.com/natcap/taskgraph/actions/runs/4736270477/jobs/8407579980?pr=87#step:3:7) was not found in the local cache
Error: Version 3.6 with arch x64 not found`

I don't think that's me, but if it is, let me know and I can work on it.

**Comment by dcdenu4** [comment]
@richpsharp, could you merge the latest changes in `main` to your branch? I'm not sure how to update the PR to reflect those changes that should help with the failing tests.

**Review by phargogh — CHANGES_REQUESTED** [review]
Thanks for digging into this @richpsharp !

Could you pull the latest changes in from `main`?  I'm hoping that will fix the build failures here (it did in #89), and I also hope that it'll cause the changes you made to the github workflow will also show up here.  We still use `rstcheck` in the test, but the interface to `rstcheck` has changed significantly, so that should also be updated when you merge in the latest changes from `main`.

File: HISTORY.rst [hunk 1]
```diff
@@ -42,6 +42,8 @@ TaskGraph Release History
   recomputation. Rather than add additional complexity around the limitations
   of this feature it is being removed to guide a design toward a standalone
   cache library if needed.
+* Fixed issue where ``TaskGraph`` ignored the parent logger level and would
+  log messages with a lower priority than its effective level.
```
[the remark this answers is not in the document]
Comment (richpsharp) [reply HISTORY.rst]: Fixed in 886e52c02fd1c7387b6070ccbe455a0fe2b5cba9

**Pull request closed without merging**